### PR TITLE
Hi more Skins added To enums :)  

### PR DESCRIPTION
**Pull request by Zam6969**
💜💜💜💜💜💜💜

### CODE REVIEW

**Comment by Zam6969** [comment]
Couldn't get the military enum to work for some reason any suggestion?

**Comment by Zam6969** [comment]
Can pull this couldn't figure this out :( commented out the code that doesn't work

File: src/servers/ZoneServer2016/managers/worldobjectmanager.ts [hunk 1]
```diff
@@ -61,6 +63,7 @@ function getRandomSkin(itemDefinitionId: number) {
     Items.HAT_BEANIE,
     Items.HAT_CAP,
     Items.HELMET_MOTORCYCLE,
+    Items.KEVLAR_DEFAULT,
   ];
   if (!allowedItems.includes(itemDefinitionId)) return itemDefinitionId;
```
Comment (0-x-2-2) [inline src/servers/ZoneServer2016/managers/worldobjectmanager.ts:68]: Why is this check even here when there is a switch statement that could just use the default clause to early return which is faster?

Comment (0-x-2-2) [reply src/servers/ZoneServer2016/managers/worldobjectmanager.ts:68]: The switch statement can be optimized by V8 to be `O(1)` or `O(log n)` if possible, currently, it is checking twice which is an extra O(n) operation that could be `O(1)` or `O(log n)`.

Comment (0-x-2-2) [reply src/servers/ZoneServer2016/managers/worldobjectmanager.ts:68]: It is also possible that this causes extra heap memory allocations but I don't know how good V8 is at doing stack allocation.

**Comment by Zam6969** [comment]
Candles!

**Review by Z1Meme — DISMISSED** [review]

**Comment by Z1Meme** [comment]
@Zam6969 is this ready?

**Comment by Zam6969** [comment]
@Z1Meme  Good?

**Review by Z1Meme — APPROVED** [review]

**Pull request merged**